kafka/src/main/java/io/specmesh/kafka/Clients.java

```diff
@@ -194,8 +194,15 @@ private static Properties getClientProperties(final String domainId, final Strin
         return adminClientProperties;
     }
 
+    /**
+     * Merges Maps
+     *
+     * @param manyMaps
+     *            - maps to merge
+     * @return merged maps
+     */
     @SuppressWarnings({"rawtypes", "unchecked"})
-    private static Map<String, Object> mergeMaps(final Map... manyMaps) {
+    public static Map<String, Object> mergeMaps(final Map... manyMaps) {
```

Comment:
Yuk!!!!  

But its a PoC

So... meh..

Reply:
yeh - i was thinking the same